src/app/services/api-users-crud.service.ts

```diff
@@ -26,11 +26,16 @@ export class ApiUsersCrudService {
 
   // Response data should be the new generated password
   public resetPassword(username: string): Observable<Response<string>> {
-    return this.serverApiService.post<string>('resetPassword', {username});
+    return this.serverApiService.post<string>('accounts/resetPassword', {username});
   }
 
   // Response data should be true if user was deleted
   public deleteUser(username: string): Observable<Response<boolean>> {
```

Comment:
Same as the above this should take in an authorityId as a param, instead of a username
```suggestion
  public deleteUser(authorityId: number): Observable<Response<boolean>> {
```